### PR TITLE
Remove template logic and use plain create extension

### PR DESCRIPTION
**Pull request by NyakudyaA**
* Remove template logic for creating databases - Only use the create extension PostGIS.
* Added an environment variable that allows creating multiple databases
* Added an environment variable that allows users to create different extensions as each database is created.
* Removed scripts that were loaded to support old versions of Postgres < 9.4. 
* Added new configs for allowing replication to work seamlessly. Especially when doing large updates.

### CODE REVIEW

**Comment by NyakudyaA** [comment]
@gubuntu can you take a look at this PR for removing templates method

File: README.md [hunk 1]
```diff
@@ -87,11 +88,12 @@ sudo docker run --name "postgis" -p 25432:5432 -d -t kartoza/postgis
 ## Environment variables
 
 You can also use the following environment variables to pass a 
-user name, password and/or default database name.
+user name, password and/or default database name(or multiple databases coma separated).
 
 * -e POSTGRES_USER=<PGUSER> 
 * -e POSTGRES_PASS=<PGPASSWORD>
 * -e POSTGRES_DBNAME=<PGDBNAME>
+* -e POSTGRES_MULTIPLE_EXTENSIONS=<postgis> # You can pass as many variables as you need.
```
Comment (gubuntu) [inline README.md]: shouldn't this be postgis not <postgis>

File: setup-database.sh [hunk 2]
```diff
@@ -35,66 +35,33 @@ su - postgres -c "${POSTGRES} -D ${DATADIR} -c config_file=${CONF} ${LOCALONLY}
...
 # Setup user
 source /setup-user.sh
 
 
 # Create a default db called 'gis' or $POSTGRES_DBNAME that you can use to get up and running quickly
 # It will be owned by the docker db user
-
-RESULT=`su - postgres -c "psql -l | grep -w ${POSTGRES_DBNAME} | wc -l"`
-echo "Check default db exists"
+# Since we now pass a coma separated list in database creation we need to search for only a single one as a test
```
Comment (gubuntu) [inline setup-database.sh]: comma not coma

**Comment by gubuntu** [comment]
LGTM - have you thoroughly tested?

maybe it's worth still loading those two legacy sql scripts

File: README.md (hunk 1 above)
Comment (NyakudyaA) [reply README.md]: @gubuntu I will fix it. The default installs these ones
`- POSTGRES_MULTIPLE_EXTENSIONS=postgis,hstore,postgis_topology`

So a user really doesn't need to pass these unless he needs to install something like `postgis_tiger_geocoder`

**Comment by NyakudyaA** [comment]
> LGTM - have you thoroughly tested?
> 
> maybe it's worth still loading those two legacy SQL scripts

I will add them back. Yes I have tested this and it works. Travis also test the replication and creating the databases

**Pull request merged**